backend/clubs/views.py

```diff
@@ -1198,9 +1199,35 @@ def create(self, request, *args, **kwargs):
             raise DRFValidationError(
                 detail="Approved activities fair events have already been created. "
                 "See above for events to edit, and "
-                f"please email {settings.FROM_EMAIL} if this is en error."
+                "please email contact@pennclubs.com if this is en error."
             )
 
+        if request.data.get("is_recurring", None) is not None:
+            parent_recurring_event = RecurringEvent.objects.create()
+            event_data = request.data.copy()
+            start_time = parse(event_data.pop("start_time"))
```

Comment:
Ideally, you would want to perform input validation on these items. This way, when the user enters something that is invalid, they get a nice error message instead of the endpoint returning a 500 and them having no idea what is going on.

This is a bit of effort, so you don't have to add it in this PR. You could redesign your API call to be a detail instead of through the main edit endpoint, which could be cleaner. In that case, you could have the detail endpoint use a custom `Serializer` (not a `ModelSerializer` since the creation is kind of weird) and use some of Django's built in parsing to make it easier for you.